configure

```diff
@@ -6306,6 +6315,8 @@ typedef $type_ptr  uintptr;
 
 #endif /* CONFIG_H */
 EOF
+copy_if_changed config.h.new config.h
+
 
 echo "Creating config.mk"
 cat > config.mk << EOF
```

Comment:
Is there a reason not to create a `config.mk.new` here and then use `copy_if_changed config.mk.new config.mk`?

Currently that does not really matter as a modified `config.mk` does not trigger any files to be rebuilt. But I am thinking this is a bug, as it causes build errors in partial rebuild due to inconsistent flags used to build files. With your changes it also means that running again `configure` with only changing options that impact this `config.mk` file (for example `--enable-asan` does not trigger any rebuild, when it should probably rebuild everything.

Reply:
No problem. I'll change it tomorrow.

Reply:
Done